lib/target.py

```diff
@@ -223,8 +311,8 @@ class Vendor:
 
 class Target:
     triple = None
-    sdk = OSType.MacOSX
-    arch = ArchType.x86_64
+    sdk = OSType.UnknownOS
+    arch = ArchType.UnknownArch
     executable_suffix = ""
     dynamic_library_prefix = "lib"
     dynamic_library_suffix = ".dylib"
```

Comment:
fair point; I agree having an opinion about a "default" target is probably incorrect in a general sense. Just was thinking that perhaps it might be good to have these as None or some other thing to avoid any potential other trip-ups